superset/assets/src/views/chartList/ChartList.less

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+.chart-list-view {
```

Comment:
Not sure this file, or `DashboardList.less`, are necessary. These styles could be moved into `ListViewStyles.less` since they're the same. 

If the chart list, dash list, or other lists are somehow different, then these files MAY be warranted, but I would actually be inclined to add a prop/class to that `ListView` component as a modifier.

Then the list view instances don't need these unique class names, and we don't have to add duplicate CSS files